gwt/src/main/java/org/geomajas/plugin/deskmanager/client/gwt/manager/datalayer/steps/EditLayerSettingsStep.java

```diff
@@ -238,4 +237,24 @@ public void reset() {
 	public boolean stepFinished() {
 		return true;
 	}
+
+	/**
+	 * Returns whether or not the specific {@link DynamicLayerConfiguration} is of a raster type of layer.
+	 * @param layerConfig
+	 * @return
```

Comment:
please complete javadoc, most of the time this is trivial.

@param layerConfig the layer configuration to check
@return true if layerConfig is a raster layer